src/signalingclient/DefaultSignalingClient.ts

```diff
@@ -91,6 +96,7 @@ export default class DefaultSignalingClient implements SignalingClient {
       clientSource: Versioning.sdkName,
       chimeSdkVersion: Versioning.sdkVersion,
     });
+    joinFrame.audioSessionId = this.audioSessionId;
```

Comment:
Have you tested that the server receives a full 64-bit integer?

Reply:
Yes.

Reply:
Nice. I think that's the first time we've ever successfully transmitted a 64-bit integer from JS to server 😄